python/tvm/driver/tvmc/autotuner.py

```diff
@@ -253,6 +321,72 @@ def get_tuning_tasks(mod, params, target, target_host=None, alter_layout=None):
     return tasks
 
 
+def autoscheduler_get_tuning_tasks(
+    mod, params, target, target_host=None, alter_layout=None, hardware_params=None
+):
+    """Get the tuning tasks for a given relay module.
```

Comment:
```suggestion
    """Get the auto-scheduler tuning tasks for a given relay module.
```
Ditto to the docstring in `autotuner_get_tuning_tasks`.